### PR TITLE
fix: align timestamp and fix ellipsis for long user and space name -MEED-260

### PR DESCRIPTION
**Pull request by GouadriaHanen**
after this change, the timestamp is aligned with the user name, and long names are now ellipsis no more hidden

### CODE REVIEW

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityMobileHead.vue [hunk 1]
```diff
@@ -8,7 +8,7 @@
       <v-list-item-title class="font-weight-bold d-flex body-2 mb-0">
         <exo-user-avatar
           :identity="posterIdentity"
-          :extra-class="'me-5'"
+          :extra-class="'me-5 text-truncate-1'"
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityMobileHead.vue]:
```suggestion
          extra-class="me-5 text-truncate"
```
1. No need to use it as a variable, you can just use static content without `:`
2. use simply text-truncate, if it doesn't wor, you should be missing a CSS `display` class

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityMobileHead.vue [hunk 2]
```diff
@@ -24,42 +24,44 @@
       </v-list-item-subtitle>
     </v-list-item-content>
   </v-list-item>
-  <v-list-item v-else class="text-truncate">
+  <v-list-item v-else class="text-truncate-1">
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityMobileHead.vue]:
```suggestion
  <v-list-item v-else class="text-truncate">
```
See above

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityMobileHead.vue [hunk 3]
```diff
@@ -24,42 +24,44 @@
...
     <exo-space-avatar
       :space="space"
+      :extra-class="'text-truncate-1'"
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityMobileHead.vue]:
```suggestion
      extra-class="text-truncate"
```
See above

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityMobileHead.vue [hunk 4]
```diff
@@ -24,42 +24,44 @@
...
       :size="42"
       avatar />
     <exo-user-avatar
       :identity="posterIdentity"
       :size="31"
       :extra-class="'ms-n4 mt-6'"
       avatar />
-    <v-list-item-content class="py-0 accountTitleLabel">
+    <v-list-item-content class="py-0 accountTitleLabel text-truncate-1">
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityMobileHead.vue]:
```suggestion
    <v-list-item-content class="py-0 accountTitleLabel text-truncate">
```
See above

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityMobileHead.vue [hunk 5]
```diff
@@ -24,42 +24,44 @@
...
       <v-list-item-title class="font-weight-bold d-flex body-2 mb-0">
         <exo-space-avatar
           :space="space"
+          :extra-class="'text-truncate-1'"
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityMobileHead.vue]:
```suggestion
          extra-class="text-truncate"
```
See above

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityMobileHead.vue [hunk 6]
```diff
@@ -24,42 +24,44 @@
...
           fullname
           bold-title
           link-style
           username-class />
       </v-list-item-title>
       <v-list-item-subtitle>
-        <v-row class="ms-2">
-          <v-col class="px-0 py-0" cols="9">
+        <v-row class="ms-2 text-truncate-1">
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityMobileHead.vue]:
```suggestion
        <v-row class="ms-2 text-truncate">
```
See above

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityMobileHead.vue [hunk 7]
```diff
@@ -24,42 +24,44 @@
...
+          <v-col class="px-0 py-0 text-truncate-1" cols="9">
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityMobileHead.vue]:
```suggestion
          <v-col class="px-0 py-0 text-truncate" cols="9">
```
See above

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityMobileHead.vue [hunk 8]
```diff
@@ -24,42 +24,44 @@
...
             <exo-user-avatar
               :identity="posterIdentity"
-              :extra-class="'text-truncate'"
+              :extra-class="'text-truncate-1'"
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityMobileHead.vue]:
```suggestion
              extra-class="text-truncate"
```
See above

**Review by AzmiTouil — APPROVED** [review]

**Pull request merged**